docs/actions.md

```diff
@@ -567,16 +570,16 @@ The action may be invoked as any other OpenWhisk action.
   }
   ```
 
-If you update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
-
+  To update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
```

Comment:
Ah I get it now. It's a guarantee then, when the user does a `wsk action update` it's the telling the system, is telling the system that any new invocations force a docker pull and pull down new `latest` even if there is a container already running with the `init` already performed.